src/NeustaConverterBundle.php

```diff
@@ -12,4 +15,14 @@ public function getPath(): string
     {
         return __DIR__;
     }
+
+    public function build(ContainerBuilder $container): void
+    {
+        parent::build($container);
+
+        $extension = $container->getExtension('neusta_converter');
+        \assert($extension instanceof NeustaConverterExtension);
```

Comment:
Kann mich mit solchen Runtime-Assertions immer wenig anfreunden, aber ist vermutlich in diesem Fall unumgänglich.

Reply:
What exactly is your problem with it?